packages/core/models/select.model.ts

```diff
@@ -0,0 +1,12 @@
+import { HTMLProps, ReactNode } from 'react'
```

Comment:
Inside `core` we need to add some shared things, this is specifically to React, use it inside React component ☺️